lib/lib-openapi/src/main/java/org/sagebionetworks/translator/ControllerToControllerModelTranslator.java

```diff
@@ -249,8 +249,11 @@ ResponseModel getResponseModel(ExecutableElement method, List<? extends DocTree>
 	 * @param blockTags the blocktags of the method
 	 * @return the description
 	 */
-	String getResponseDescription(List<? extends DocTree> blockTags) {
+	String getResponseDescription(List<? extends DocTree> blockTags, ExecutableElement method) {
 		Optional<String> returnComment = getReturnComment(blockTags);
+		if (returnComment.isEmpty() && method.getReturnType().getKind().equals(TypeKind.VOID)) {
+			return "The request was successful, but there is no response content.";
```

Comment:
This will never return null.
For the void case return "Void" for the default case return "Auto-generated description..."
Can use Optional.getOrElse() instead of the java tertiary.